src/main/java/com/jitterted/mobreg/adapter/in/web/member/EnsembleSummaryView.java

```diff
@@ -16,24 +20,26 @@ public record EnsembleSummaryView(long id,
                                   String googleCalendarLink,
                                   int numberRegistered,
                                   String recordingLink,
-                                  String memberStatus) {
+                                  String memberStatus,
+                                  List<MemberView> acceptedMembers) {
 
-    public static List<EnsembleSummaryView> from(List<Ensemble> ensembles, MemberId memberId) {
+    public static List<EnsembleSummaryView> from(List<Ensemble> ensembles, MemberId memberId, List<Member> allUsers) {
```

Comment:
I think it'd be better to pass in the `MemberService` and let the code "pull" the `Member` objects it needs instead of passing _all_ members. See src/main/java/com/jitterted/mobreg/adapter/in/web/admin/EnsembleDetailView.java for how I did this.

Reply:
Alright, but just to make sure: you're not worried about performance of making a DB query to pull info for each member in each individual ensemble? (many will be repeated calls to get info about the same member, and it seems easy enough to avoid)
This call gets information for all the mobs to show on the page:
`List<EnsembleSummaryView> from(List<Ensemble> ensembles, MemberId memberId, List<Member> allUsers)`
and I think will result in a lot of duplicate calls to get information about the same member, and it will get worse with time?

Reply:
> worried about performance of making a DB query to pull info for each member in each individual ensemble

I would add caching in that case.